src/systemd/sd-dhcp-client.h

```diff
@@ -160,7 +160,7 @@ sd_dhcp_client *sd_dhcp_client_unref(sd_dhcp_client *client);
 
 /* NOTE: anonymize parameter is used to initialize PRL memory with different
  * options when using RFC7844 Anonymity Profiles */
-int sd_dhcp_client_new(sd_dhcp_client **ret, int anonymize);
+__extension__ int sd_dhcp_client_new(sd_dhcp_client **ret, int anonymize, bool rapid_commit);
```

Comment:
hmm?

Reply:
Apparently I need this attribute to use `bool`, and the param is so that I know to put the rapid commit flag in the discover message. Is there a better way?